core/pkg/sync/grpc/grpc_sync_test.go

```diff
@@ -677,3 +661,14 @@ func (b *bufferedServer) SyncFlags(_ *v1.SyncFlagsRequest, stream syncv1grpc.Fla
 func (b *bufferedServer) FetchAllFlags(_ context.Context, _ *v1.FetchAllFlagsRequest) (*v1.FetchAllFlagsResponse, error) {
 	return b.fetchAllFlagsResponse, b.fetchAllFlagsError
 }
+
+func (b *bufferedServer) GetMetadata(_ context.Context, _ *v1.GetMetadataRequest) (*v1.GetMetadataResponse, error) {
+	return &v1.GetMetadataResponse{
```

Comment:
not sure what would be meaningful here

Reply:
I think we can leave it empty for now.
Eventually we will decide how to populate it.

Reply:
@Kavindu-Dodan @RealAnna I've pushed an update which uses a slightly new (breaking) proto. It changes the `metadata` type to a Struct, which is optional, so now the `GetMetadataResponse` is completely empty.

Reply:
Thanks @toddbaert  Right now we do not expose any metadata, but this is a good move :)